src/dscanner/analysis/useless_initializer.d

```diff
@@ -361,6 +364,33 @@ public:
 		NotKnown nk = NotKnown.init;
 	}, sac);
 
+	// passes
+	assertAnalyzerWarnings(q{
+			@("nolint(useless_initializer)")
+			int a = 0;
+		    ubyte a = 0x0;      /+
+		              ^^^ [warn]: X +/
```

Comment:
```suggestion
			int a = 0; /+
			        ^ [warn]: X +/
```

try not changing here too much / being too smart, otherwise others reading this code might think this error comes from using 0x0 instead of 0 or only applies to ubyte and not int.